### ISSUE TITLE
Error: `apply_removal_to_simple_attribute': undefined method `-' for "":String (NoMethodError)

### ISSUE THREAD

**Issue by dtamiazzo** [Lightricks/Kintsugi#44]
Hi, I'm trying to resolve a complicated merge and I'm getting the following error:

```
/usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:189:in `apply_removal_to_simple_attribute': undefined method `-' for "":String (NoMethodError)
Did you mean?  -@
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:170:in `simple_attribute_value_with_change'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:180:in `block in simple_attribute_value_with_change'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:177:in `each'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:177:in `simple_attribute_value_with_change'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:162:in `apply_change_to_simple_attribute'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:53:in `apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:82:in `block in apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `each'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:82:in `block in apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `each'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:82:in `block in apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `each'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:82:in `block in apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `each'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:81:in `apply_change_to_component'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/apply_change_to_project.rb:40:in `apply_change_to_project'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi.rb:92:in `apply_change_and_copy_to_original_path'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi.rb:44:in `resolve_conflicts'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/lib/kintsugi/cli.rb:195:in `block in create_root_command'
	from /usr/local/lib/ruby/gems/3.0.0/gems/kintsugi-0.4.3/bin/kintsugi:34:in `<top (required)>'
	from /usr/local/lib/ruby/gems/3.0.0/bin/kintsugi:23:in `load'
	from /usr/local/lib/ruby/gems/3.0.0/bin/kintsugi:23:in `<main>'
```
	
Any idea why? Thanks

**Comment by ashdnazg** [comment]
Thanks for the report!
Is it perhaps possible for you to share the two branches with us?

**Comment by byohay** [comment]
@dtamiazzo I believe I solved it in #45
